### ISSUE TITLE
NG+ Auto-Connect

### ISSUE THREAD

**Issue by TheYellowArchitect** [TheYellowArchitect/doubledamnation#32]
There should be an automatic reconnection after the NG+ cutscene (final scene)
		
Otherwise, players finish game, and then they will chat a little, and exit the game, as they see they have disconnected and their inputs are fully local, and would have to reconnect.
In short, this subtle action of disconnection is like saying "We are done here. The game is finished, there is nothing else for you here."
		
If the players are connected after NG+, then a simple flail of the joystick or any keyboard input, will signal to the other player that they want to play, or could play a bit more - a signal that NG+ is a thing!
In short, to try different endings, and the gameplay will be far different and swifter since they have mastered the game. Could reach The End again within 10 minutes!
But if they are disconnected, there isnt even a subtle hint for this.
	
**Code**
See the following variables of playerCoreStats which are saved as JSON
`bool playedAsHost`
`bool playedAsClient`
`ulong steamLobbyId`
`ulong steamFriendId`

So, at the start of the game, the above 4 values are parsed from the latest JSON, and then host/connection logic happens.
